src/redux/actions/fetching.js

```diff
@@ -0,0 +1,18 @@
+import { START_FETCHING, STOP_FETCHING } from './actionTypes';
+
+
+export const startFetching = () => (
```

Comment:
@chingsley what does this file do?

Reply:
I use it to control the loader. When I make a request to the database, I will set the state property `fetching` to true by dispatching the START_FETCHING action type. When `fetching` is true, the `loader` (the round balls will show up showing the user that an action is being performed in the background). The reverse is the case for STOP_FETCHING - to stop the loader.

Reply:
I thought this is already in use? How are others achieveing it?

Reply:
I created the loader. I created the loader. Others did not use loader. I just asked.